book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_reader :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Dear @blueberry1312. I like how you implement the add_rental method in your book Cleass. But we need to guarantee a has-many relationship here. which means all the rentals that are added should belong to this book. Currently from your implementation, if I pass a rental for a different book, It will work fine (as long as I manage to pass a rental). This is not the expected behavior. the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

> Something like 
```ruby
def add_rental(person, date)
   Rental.new(date, self, person)
  end
```

This creates a rental, in turn, creating a new rental will do a `book.rentals << self`. this asserts the `'has-many'` behavior of our association.